apps/renderer/src/index.css

```
@@ -179,19 +180,35 @@ body {
+::-webkit-scrollbar-thumb {
+  background: var(--color-border);
+  border-radius: 4px;
+}
+::-webkit-scrollbar-thumb:hover {
+  background: var(--color-accent);
+}
+
 .toc-item {
-  @apply block w-full border-l-2 border-transparent py-1 text-left text-sm transition-colors;
+  @apply block w-full rounded-md py-1.5 text-left text-sm transition-all duration-150;
   color: var(--color-text-muted);
```

Stylelint (17.13.0)

[error] Expected empty line before declaration (declaration-empty-line-before)

(declaration-empty-line-before)